usage.md

```diff
@@ -0,0 +1,52 @@
+The attestation library is available under the [attestation-dev branch](https://github.com/hyperledger-labs/private-data-objects/tree/attestation-dev).
+
+The library is divided in 3 main components:
+1. attestation generation, inside the enclave
+2. attestation conversion to evidence, outside of the enclave
+3. evidence verification, inside or outside of the enclave
+
+![image](https://github.com/user-attachments/assets/d3042b3a-37dd-4966-9c8a-9561e93f6174)
+
+The figure depicts the flows for attestation generation, conversion and verification.
+For additional technical details look at the `test/attested_evidence_test.sh` script in the library.
+Also, for additional details about the APIs look at the headers in the `include` folder.
+
+**Attestation generation.**
+The library exposes the `init_attestation(params)` and `attestation_blob = get_attestation(statement)` APIs.
+`params` is a json blob to initialize the internal state of the library for an attestation. The `statement` is a binary string which (in SGX) is hashed to generate the "report data" to be attested.
+```jsonc
```

Comment:
Question: instead of runtime selection of the attestation method, can this just be compile time (or link time)? Make each method a separate library that can be linked. It seems unlikely that the attestation method would change for a particular application *at run time*.